core-assemblies/core-features/src/main/feature/feature.xml

```diff
@@ -142,6 +142,7 @@
   <feature name="apromore-portal" version="${project.version}" description="Apromore core web browser interface">
     <feature>apromore-calendar</feature>
     <feature>apromore-manager</feature>
+    <feature>jms</feature>
```

Comment:
This fixes a bug the integration test picked up.  Previously the apromore-portal feature wouldn't start on its own due to a missing JMS API class.